python/tank/templatekey.py

```diff
@@ -305,8 +304,10 @@ def __init__(self,
         :param shotgun_entity_type: For keys directly linked to a shotgun field, the entity type.
         :param shotgun_field_name: For keys directly linked to a shotgun field, the field name.
         :param exclusions: List of forbidden values.
-        :param abstract: Bool, should this key be treated as abstract.
-        :param length: int, should this key be fixed length
+        :param abstract: Flagging that this should be treated as an abstract key.
```

Comment:
[minor] Do we want to add param types in here while we're at it?

`:param bool abstract: ...`